### ISSUE TITLE
Icon for Computed Reference Link

### ISSUE THREAD

**Issue by nrshapiro** [Psychoanalytic-Electronic-Publishing/PEP-Web-User-Interface#296]
I'm not a fan of the current icon for the computed reference link.  It took me a bit to figure out what it was, and now that I know, the icon just strikes me as unprofessional looking, and also, not easily associated with the info icon.

![image](https://user-images.githubusercontent.com/4837554/108860591-58db4a80-75bc-11eb-8a06-6caa1397a0bc.png)

On our old site, we use:

![image](https://user-images.githubusercontent.com/4837554/108856017-9b4e5880-75b7-11eb-9133-1f2754e680eb.png)

We could use something similar, but use AI instead of related, since in fact, often it's actually just the right link, but not for sure, since it's automatically linked.

e.g., AI(i)-> and -> for links, where -> is a proper arrow like you used, and i is the info icon that has hover text that explains. 

I've also experimented with a graphical representation, and we could always get our graphic artist to do a better and more creative job than I did here (AIMagicWand.svg, but I had to convert to JPG to display here):

<img width="28" alt="2021-02-23_09h37_10" src="https://user-images.githubusercontent.com/4837554/108858930-b79fc480-75ba-11eb-8817-421dff6351db.png">

There may also be a better icon in the font awesome set you are using, I'd be happy to see other suggestions.

**Comment by nrshapiro** [comment]
I think we should go with [AI Linked (i)->] and [->] for the short term.  I think the brackets will help with this version, particlarly the long form as they do in the old GVPi interface.  We'll look at icons when we're less busy.  Use arrows and the info icon where I used chars above.  I will ask our GD later, when we have more than one little task to do!

One other important note: where rxp and rx are both included in a reference, check if the values are the same--if they are, just show the rx link.  If they are different, show both.  The next time I reprocess the whole set, I will try to eliminate the rxp if they are the same, but that may not be for a quite a few months.

**Comment by nrshapiro** [comment]
Link with examples since I left them out before:

https://pep-web.rocks/search/document/TVPA.001.0036A?searchTerms=%5B%7B"type"%3A"reference"%2C"term"%3A"compound%20cinema"%7D%5D

**Comment by bakerac4** [comment]
@nrshapiro 

https://fontawesome.com/icons/laptop-code?style=solid
https://fontawesome.com/icons/computer-classic?style=solid
https://fontawesome.com/icons/file-code?style=solid
https://fontawesome.com/icons/head-side-brain?style=solid

Here are a few different options that might work. Let me know your favorite

**Comment by nrshapiro** [comment]
@bakerac4  

Thanks for your suggestions.

I ran this by David with some options that I thought could potentially work (including the head-side-brain).  I used Photoshop to simulate them at the smaller size along with some references.  

One error: I also put in the regular link, with a new icon.  I was thinking they could both be present sometimes, so I put them together to demonstrate.  Certainly the XML would support that, but I don't think my postprocessing will add the Related Link when there is a matching link.

1) He picked the link icon you see on the second to last, Benjamin, J. 1995b.  I think that can work too.  Just find Link in font awesome. 

2) I also used a (?) instead of (i).  That seems more consistent with the rest of the interface.

3) And lastly, I changed to a graphical arrow as you see.  Also easily found in Font Awesome, mainly because it looked better with the graphical link.  

![image](https://user-images.githubusercontent.com/4837554/112074650-25ed8d80-8b4d-11eb-9723-31fd18fb428d.png)
